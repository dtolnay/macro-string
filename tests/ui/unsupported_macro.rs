use macro_string_eval::eval;

const _: &str = eval!(include_bytes!("..."));

fn main() {}
