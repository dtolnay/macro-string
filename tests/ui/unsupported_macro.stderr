error: expected one of: string literal, `concat`, `env`, `include`, `include_str`, `stringify`
 --> tests/ui/unsupported_macro.rs:3:23
  |
3 | const _: &str = eval!(include_bytes!("..."));
  |                       ^^^^^^^^^^^^^
